### ISSUE TITLE
Preload hedgehog images

### ISSUE THREAD

**Issue by Aeolun** [PostHog/posthog#25917]
## Bug description

Any time the hedgehog switches animations, there is a brief flash where they are invisible as the required image is loaded in. 

## How to reproduce

1. Enable hedgehog mode
2. Observe hedgehog long enough for them to switch animations (preferably don't do on local, or enable 3g mode to make it more clear or something)

## Visual

https://github.com/user-attachments/assets/e69f3fe1-6f30-4c20-a49d-def0091c49d4

## Additional context

This is obviously not high priority, but for such a fun feature it's a bit sad it immediately gives such a half baked impression.

### Debug info

```shell
- [ ] PostHog Cloud, Debug information: [please copy/paste from https://us.posthog.com/settings/project-details#variables or https://eu.posthog.com/settings/project-details#variables]
- [x] PostHog Hobby self-hosted with `docker compose`, version/commit: [13f48d74d07e8e925f73f7e0c5ae59091372e64f]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see [`Sunsetting Kubernetes support`](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]
```

**Comment by Aeolun** [comment]
Huh, just noticed that even on a self hosted installation these images get loaded from the US static CDN, which probably has something to do with the noticeable delay.

**Comment by Twixes** [comment]
This is important. Thank you for highlighting. :) Fix here: #26027
